package.json

```diff
@@ -18,9 +18,8 @@
     "prepublish": "npm run build",
     "prepublish-watch": "npm run watch",
     "start": "./node_modules/.bin/webpack-dev-server",
-    "tap-integration": "./node_modules/.bin/tap ./test/integration/*.js",
-    "tap-unit": "./node_modules/.bin/tap ./test/unit/*.js",
-    "test": "npm run lint && npm run tap-unit && npm run tap-integration",
+    "tap": "./node_modules/.bin/tap ./test/{unit,integration}/*.js",
```

Comment:
I had broken this out in case the `{unit,integration}` syntax breaks on Windows. Not sure if that's the case or not.